AndroidAnnotations/functional-test-1-5/src/main/java/org/androidannotations/test15/ActivityWithServices.java

```diff
@@ -121,6 +125,9 @@
 	@SystemService
 	DropBoxManager dropBoxManager;
 	
+	// @SystemService
+	// EthernetManager ethernetManager; // hidden API
```

Comment:
No, these services are intended for custom Android builds. See the [release notes](https://github.com/excilys/androidannotations/wiki/ReleaseNotes#22):

 > and even hidden system services (useful for custom Android builds)

Actually these Services classes are not available in the normal android.jar at compile time, so this only works if you are building against the AOSP tree or you create a custom internal jar.